src/cryptoadvance/specter/server_endpoints/wallets.py

```diff
@@ -73,7 +73,7 @@ def wrapper(*args, **kwargs):
 @login_required
 def wallets_overview():
     app.specter.check_blockheight()
-    for wallet in app.specter.wallet_manager.wallets.values():
+    for wallet in list(app.specter.wallet_manager.wallets.values()):
```

Comment:
why is this needed? wallet_manager.wallets is a dict (https://github.com/cryptoadvance/specter-desktop/blob/2baa83f572ed59e8d41ced4bc8792cc739d47998/src/cryptoadvance/specter/wallet_manager.py#L54) so shouldnt `values()` always return a valid list?

Reply:
Yes it is a valid list, but since wallets are being loaded asynchronously, the length of the list could change during iteration, which will cause an error. This just clones the list so it won't be affected by async changes in the dictionary.